StripeIdentity/StripeIdentity/Source/NativeComponents/Views/PhoneOtpView.swift

```diff
@@ -112,12 +112,12 @@ class PhoneOtpView: UIView {
             submittingIndicator.isHidden = true
             submittingIndicator.stopAnimating()
         case .SubmittingOTP:
-            otpTextField.isEnabled = true
+            otpTextField.isEnabled = false
             otpErrorField.isHidden = true
             submittingIndicator.isHidden = false
             submittingIndicator.startAnimating()
         case .ErrorOTP:
-            otpTextField.value = ""
+            otpTextField.performInvalidCodeAnimation()
```

Comment:
I guess this is a new change in behavior where an invalid input will preserve the OTP code? Is this coming from design feedback?